test/index.js

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env node
+'use strict';
+// 'use strict' is here so we can use let and const in node 4
```

Comment:
Delete this line?

Reply:
'use strict' is required for block scope let and const in node v4. I figured I would write a comment so people don't remove it since it isn't needed in newer versions

Reply:
It throws an error when you remove it, right? So if someone removes it the tests should fail (I assume).

Reply:
It throws an error that might not be obvious that they have to put `'use script';` back.

I also like the comment because it lets us know that `'use script';` can be removed once we no longer support testing on node 4 .

Reply:
Sounds good to me 👍